src/NServiceBus.AcceptanceTests/Sagas/When_saga_id_changed.cs

```diff
@@ -28,7 +28,12 @@ await Scenario.Define<Context>()
                 .AllowExceptions()
                 .Done(c => c.MessageFailed)
                 .Repeat(r => r.For(Transports.Default))
```

Comment:
Should we start converting test to not user .Repeat where possible? (or not in scope?)